src/firejail/usage.c

```diff
@@ -32,6 +32,7 @@ static char *usage_str =
 	"    --allusers - all user home directories are visible inside the sandbox.\n"
 	"    --apparmor - enable AppArmor confinement.\n"
 	"    --apparmor.print=name|pid - print apparmor status.\n"
+	"    --apparmor=profile - enable AppArmor confinement with a certain profile.\n"
```

Comment:
```suggestion
	"    --apparmor - enable AppArmor confinement with the default profile.\n"
	"    --apparmor=profile_name - enable AppArmor confinement with a custom profile.\n"
	"    --apparmor.print=name|pid - print apparmor status.\n"
```

Change order and clarify usage.

Usually, a `foo=bar` option should appear right after the `foo` option.